pkg/api/tag_test.go

```diff
@@ -410,9 +410,9 @@ func TestTags(t *testing.T) {
 	})
 }
 
-// isTagFoundInResponse verifies that the tag id is found in the supplied HTTP headers
+// IsTagFoundInResponse verifies that the tag id is found in the supplied HTTP headers
 // if an API tag response is supplied, it also verifies that it contains an id which matches the headers
-func isTagFoundInResponse(t *testing.T, headers http.Header, tr *api.TagResponse) uint32 {
+func IsTagFoundInResponse(t *testing.T, headers http.Header, tr *api.TagResponse) uint32 {
```

Comment:
why is it needed to export this function? it is in the same namespace as the tests

Reply:
Sorry about that. I originally did not notice that they are in the same package and then forgot to revert that. Not I reverted it.